Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -37,8 +37,23 @@ using tchar = char;
 using tstring = std::string;
 #endif
 
+#ifdef __linux__
```

Comment:
no macro LINUX. It is __linux__
http://stackoverflow.com/questions/6649936/c-compiling-on-windows-and-linux-ifdef-switch
or
http://sourceforge.net/p/predef/wiki/OperatingSystems/